lizard_efcis/import_data.py

```diff
@@ -154,7 +154,7 @@ def _get_foreignkey_inst(
                     logger.info("get wns %s %s" % (
                         datatype, ''.join(val_raw.split(' '))))
                 inst = class_inst.objects.get(
-                    **{'wns_oms_space_less__iexact': ''.join(val_raw.split(' '))})
+                    wns_oms__iexact=''.join(val_raw.split(' ')))
```

Comment:
Ter info: `some_method(**{'key': 'value})` is identiek aan `some_method(key='value')`. Die dubbele ster zorgt er alleen voor dat de dictionary wordt "uitgepakt" als keyword arguments. Idem met zoiets als `some_method(*args)`, dat pakt "args" als een lijst uit.